docker/nginx/conf.d/server/server.api

```diff
@@ -348,6 +348,42 @@ location ~ "^/file/(([a-zA-Z0-9-_]{46}|[a-z0-9]{55})(/.*)?)$" {
     include /etc/nginx/conf.d/include/location-skylink;
 }
 
+location /skynet/trustless/basesector {
+    include /etc/nginx/conf.d/include/cors;
+    include /etc/nginx/conf.d/include/proxy-buffer;
+    include /etc/nginx/conf.d/include/track-download;
+
+    limit_conn downloads_by_ip 10; # ddos protection: max 100 downloads at a time
```

Comment:
100 or 10?

Reply:
good catch, bad copy & paste, I played with this number locally and forgot to revert it

Reply:
```suggestion
    limit_conn downloads_by_ip 100; # ddos protection: max 100 downloads at a time
```